src/Server/middleware/bootstrapSharifyAndContextLocalsMiddleware.ts

```diff
@@ -15,7 +15,9 @@ export function bootstrapSharifyAndContextLocalsMiddleware(
 ) {
   const ua = req.get("user-agent") || ""
 
-  updateSharifyAndContext(res, "CURRENT_PATH", parse(req.url).pathname)
+  const parsed = parse(req.url)
+  updateSharifyAndContext(res, "CURRENT_PATH", parsed.pathname)
+  updateSharifyAndContext(res, "CURRENT_SEARCH", parsed.search)
```

Comment:
is it possible to rename this to something more verbose, like `CURRENT_SEARCH_PARAMS` or `CURRENT_QUERY_PARAMS`? From the outside, it looks like something related to "search", and lots of devs aren't aware that search = query params

Reply:
Its just that its common to pass over actual search stuff from the server to the client and so on. Disambiguating this would be helpful. `CURRENT_LOCATION_SEARCH`, though params communicates something more effectively, even if it needs to be parsed.

Reply:
I'd be more inclined to just stick a fully qualified URL in here and then we just parse it on the client.

I can just refactor this hook to not run on the server though. I'll do that instead.

Reply:
Either way 😄. Ambiguous ENV vars placed in sharify, etc, cause trouble through time, so as long as that's clarified a bit and / or removed.